### DISCUSSION TITLE
`aconfmgr diff` should exit with 1 if there is a diff

### DISCUSSION THREAD

**Discussion by infokiller** [CyberShadow/aconfmgr#128] [Ideas]
I think it would make sense to replicate the behavior of `diff`. My use case is to run `aconfmgr diff` periodically and notify me when the system diverges from my config.
For context, my overall goal is to automate archlinux updates and maintenance as much as possible, and notify me when manual intervention is needed (new arch news, pacnew files, etc.).

**Comment by CyberShadow** [comment]
`aconfmgr diff` is for files only (and requires that a path is specified). I think your goal would be better achieved by running `aconfmgr save`, and then checking the existence of `99-unsorted.sh`.

**Reply by infokiller** [reply]
Thanks, I'll try to do it this way
